test/button-state.test.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
```

Comment:
@marclove  FYI we briefly discussed this at standup. I think we should merge this in without the ButtonState tests and see how it goes with the Select Menu divided between two test suites. If we decide it's overkill we can remove mocha for browser modules. What do you think?